app/cdap/cdap.html

```diff
@@ -16,25 +16,55 @@
 
 <!DOCTYPE html>
 <html lang="en">
-<head>
+  <head>
```

Comment:
Are changes to this file necessary? Can you revert them if not?

Reply:
We have removed the changes made to this file.
If any changes are there, they might just be lining or formatting changes.

We have used the original font family as discussed in the daily status call with Nick and Sebastian.